tst/fakepkg/doc/make_doc

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+echo "TeXing documentation"
+# TeX the manual 
+tex manual
+# TeX the manual again to incorporate the ToC ... and build the index
+tex manual; ../../../doc/manualindex manual
+# Finally TeX the manual again to get cross-references right
+tex manual
+# Create PDF version
+pdftex manual; pdftex manual
```

Comment:
Is that PDF version used for anything? I'd much rather *not* add that manual.pdf to the repository, as it is a binary blob.

Reply:
It's mentioned in PackageInfo.g and its used for its validation. But it's fine to not to commit the PDF produced by `make_doc` (will describe that in some README), since it is not needed for _this_ test.

Reply:
I would recommend this then: remove the `pdftex` calls here, and replace `manual.pdf` by a plain text file saying something like
> This is a placeholder, to enable validation of this mock package, while avoiding the need to add a binary file to the repository.

Reply:
Or, if you really want to keep these two lines (but why?), then add the `manual.pdf` to a `.gitignore` file, to prevent people from adding it accidentally in the future.

Note that you can have a local `.gitignore` file inside `tst/fakepkg/` or even `tst/fakepkg/doc/`

Reply:
I like the idea of plain text `manual.pdf` then. I don't really need to produce pdf - I can judge whether the manual is produced from the dvi file.